src/camera.cpp

```diff
@@ -453,12 +452,13 @@ void Camera::update(LocalPlayer* player, f32 frametime, f32 busytime, f32 tool_r
 
 	// Get FOV
 	f32 fov_degrees;
-	if (player->getPlayerControl().zoom && player->getCanZoom()) {
-		fov_degrees = m_cache_zoom_fov;
+	// Disable zoom with zoom FOV = 0
+	if (player->getPlayerControl().zoom && player->getZoomFOV() != 0.0f) {
```

Comment:
perhaps change this to `player->getZoomFOV() > 0.001f` for floating point errors and invalid input from Lua-side (as negative values sometimes disable other features)

Reply:
Good idea.